### PR TITLE
online method missing self arg

### LINKED ISSUE #356: TypeError: is_online() takes 0 positional arguments but 1 was given
getting this on a telliot instance running on AWS with start command:
`telliot-feeds -a mumbai2 report -p YOLO -wp 120`

```
ERROR   | telliot_core | Exception occurred in telliot-core app
ERROR   | telliot_core | <class 'TypeError'>
ERROR   | telliot_core | is_online() takes 0 positional arguments but 1 was given
ERROR   | telliot_core | ['  File "/home/ubuntu/telliot-feeds/src/telliot_feeds/cli/commands/report.py", line 499, in report\n    await reporter.report()\n', '  File "/home/ubuntu/telliot-feeds/src/telliot_feeds/reporters/tellorflex.py", line 365, in report\n    online = self.is_online()\n']
Traceback (most recent call last):
  File "/home/ubuntu/tenv/bin/telliot-feeds", line 8, in <module>
    sys.exit(main())
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/home/ubuntu/tenv/lib/python3.9/site-packages/telliot_core/cli/utils.py", line 18, in wrapper
    return asyncio.run(f(*args, **kwargs))
  File "/usr/lib/python3.9/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/home/ubuntu/telliot-feeds/src/telliot_feeds/cli/commands/report.py", line 499, in report
    await reporter.report()
  File "/home/ubuntu/telliot-feeds/src/telliot_feeds/reporters/tellorflex.py", line 365, in report
    online = self.is_online()
TypeError: is_online() takes 0 positional arguments but 1 was given
```

### PR DESCRIPTION
**Pull request by akremstudy**
### Summary
Minor fix: telliot was breaking because a method in interval.py was not given the "self" arg
https://github.com/tellor-io/telliot-feeds/blob/5ce625a0062f14b787923bf54b6a7c4bd4554c5d/src/telliot_feeds/reporters/interval.py#L295
Closes #356 
### Steps Taken to QA Changes
submitted txn using telliot: https://mumbai.polygonscan.com//tx/0x38cc45e8ea8b96bb92797c8277c1389b4749ae888a1e058ea6a91a8d96eb9eeb

### Checklist

This pull request is:

- [ ] A documentation error, docs update, or typographical error fix
	- No tests or issue needed
- [x] A code fix
	- Please reference the related issue by including "Closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a bug report issue 
	- Please include tests. Fixes without tests will not be accepted unless it's related to the documentation only.
    - Please make sure docs are updated if need be
- [ ] A feature implementation
	- Please reference the related issue by including "Closes `<link to issue>`" in this Pull Request's summary section. 
        - If no issue exists, please create a feature enhancement issue 
	- Please include tests
    - Please make sure docs updates are both thorough and easy to reproduce by somebody with limited knowledge of the feature that you are submitting

**Happy engineering!**

### CODE REVIEW

File: src/telliot_feeds/reporters/tellorflex.py [hunk 1]
```diff
@@ -362,7 +362,7 @@ async def report(self) -> None:
         """Submit latest values to the TellorFlex oracle."""
 
         while True:
-            online = self.is_online()
+            online = await self.is_online()
```
Comment (tallywiesenberg) [inline src/telliot_feeds/reporters/tellorflex.py:365]: are you sure you should add an `await` here?

Comment (akremstudy) [reply src/telliot_feeds/reporters/tellorflex.py:365]: i believe so cause its an async function

Comment (tallywiesenberg) [reply src/telliot_feeds/reporters/tellorflex.py:365]: also don't forget `tox  -e style` :)

Comment (akremstudy) [reply src/telliot_feeds/reporters/tellorflex.py:365]: didn't forget

Comment (akremstudy) [reply src/telliot_feeds/reporters/tellorflex.py:365]: its unrelated to this, its being caused by something else

Comment (tallywiesenberg) [reply src/telliot_feeds/reporters/tellorflex.py:365]: oh right. i ran into that too

Comment (akremstudy) [reply src/telliot_feeds/reporters/tellorflex.py:365]: src/telliot_feeds/sources/price_aggregator.py:21:1: B024 PriceAggregator is an abstract base class, but it has no abstract methods. Remember to use @abstractmethod, @abstractclassmethod and/or @abstractproperty decorators.

**Review by oraclown — APPROVED** [review]
Thanks @akremstudy

**Pull request merged**